lib/services/s3.d.ts

```diff
@@ -1,6 +1,67 @@
 import {Service} from '../service';
 import {ManagedUpload} from '../s3/managed_upload';
 import S3 = require('../../clients/s3');
+
+export interface PresignedPostParams {
```

Comment:
Typically after importing `S3` in TypeScript I can do something like this:

```
import { S3 } from 'aws-sdk';
const params: S3.Types.PresignedPostParams = {
  // ...
};
```

Unfortunately, it seems that the `PresignedPostParams` are exposed this way (likewise `PresignedPost`).